specification/resources/resource-manager/Microsoft.Resources/stable/2016-06-01/subscriptions.json

```diff
@@ -44,14 +44,10 @@
         "description": "This operation provides all the locations that are available for resource providers; however, each resource provider may support a subset of this list.",
         "parameters": [
           {
-            "name": "subscriptionId",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The ID of the target subscription."
+            "$ref": "#/parameters/ApiVersionParameter"
           },
```

Comment:
@vivsriaus the order of parameters does matter for some SDKs. Please, revert back to
- the api version should be the first parameter,
- the subscription is should be the second parameter.

Reply:
@sergey-shandar Fixed